cluster/gce/configure-vm.sh

```diff
@@ -529,6 +529,12 @@ function salt-set-apiserver() {
 EOF
 }
 
+function salt-set-test-node() {
+  cat <<EOF >>/etc/salt/minion.d/grains.conf
+  test: true
```

Comment:
This is how it was done for other things (e.g. salt-set-apiserver), so I just went the same way. @zmerlynn ?